packages/fxa-settings/src/pages/Signup/index.tsx

```diff
@@ -0,0 +1,490 @@
+            </button>
+          </FtlMsg>
+        </div>
+      )}
+
+      <div className="mt-4 mb-6">
+        <p className="break-all">{email}</p>
+
+        {canChangeEmail && (
+          <FtlMsg id="signup-change-email-link">
+            <a href="/" className="link-blue text-sm">
+              Change email
+            </a>
+          </FtlMsg>
+        )}
+      </div>
+
+      <FormPasswordWithBalloons
+        {...{
+          formState,
+          errors,
+          trigger,
+          register,
+          getValues,
+          newPasswordErrorText,
+          setNewPasswordErrorText,
+          onFocus,
+          email,
+          newPasswordLabel,
+          confirmPasswordLabel,
+          submitButtonText,
+          onFocusMetricsEvent,
+        }}
+        onSubmit={handleSubmit(onSubmit)}
+        loading={false}
+        showConfirmPwdBalloon
+      >
+        {/* Everything below gets passed to FormPassword as children.
+            We might want to review how this is handled */}
+        {/* TODO: improve a11y - original component had a SR-only label
+        "How old are you? To learn why we ask for your age, follow the “why do we ask” link below. */}
+        <InputText
+          name="userAge"
+          label={ageCheckLabel}
+          inputMode="numeric"
+          className="text-start"
+          pattern="[0-9]*"
+          maxLength={3}
+          onChange={(e) => {
+            // TODO: allow only digits in input field
+            setAge(parseInt(e.target.value));
+            // clear error tooltip if user types in the field
+            if (ageCheckErrorText) {
+              setAgeCheckErrorText('');
+            }
+          }}
+          onBlurCb={onBlurAgeInput}
+          errorText={ageCheckErrorText}
+          anchorStart
+          required
+        />
+        <FtlMsg id="signup-coppa-check-explanation-link">
+          <LinkExternal
+            href="https://www.ftc.gov/business-guidance/resources/childrens-online-privacy-protection-rule-not-just-kids-sites"
+            className="link-blue text-start text-sm mb-4"
+          >
+            Why do we ask?
+          </LinkExternal>
+        </FtlMsg>
+
+        {isCWTSEnabled && (
+          <>
+            <FtlMsg id="signup-choose-what-to-sync-heading">
+              <h2 className="text-base">Choose what to sync</h2>
+            </FtlMsg>
+            <div className="flex flex-wrap text-start ltr:mobileLandscape:ml-6 rtl:mobileLandscape:mr-6 mb-1 text-sm">
+              {engines.map((engine) => (
+                <div
+                  key={engine.id}
+                  className="mb-4 relative flex-50% rtl:mobileLandscape:pr-6 ltr:mobileLandscape:pl-6 rtl:pr-3 ltr:pl-3 flex items-center"
+                >
+                  <Checkbox
+                    label={engine.text}
+                    prefixDataTestId={engine.id}
+                    defaultChecked={engine.checked}
+                  />
+                </div>
```

Comment:
Smidge too much space between these checkboxes and "Create account".

Prod:
![image](https://user-images.githubusercontent.com/13018240/214702720-c4f28596-feb1-43a1-b2df-ed0742a01881.png)

vs Storybook:
<img width="550" alt="image" src="https://user-images.githubusercontent.com/13018240/214702768-471d4d3c-b1c2-4f6f-a286-a21cd36e5f1e.png">

I personally don't mind the extra space since it creates some clickable space though 🤷‍♀️